ci/gpu/build.sh

```diff
@@ -59,8 +59,8 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
 # Install the master version of dask, distributed, and dask-ml
 gpuci_logger "Install the master version of dask and distributed"
 set -x
-pip install "git+https://github.com/dask/distributed.git" --upgrade --no-deps
-pip install "git+https://github.com/dask/dask.git" --upgrade --no-deps
+pip install "git+https://github.com/dask/distributed.git@2021.05.1" --upgrade --no-deps
+pip install "git+https://github.com/dask/dask.git@2021.05.1" --upgrade --no-deps
```

Comment:
```suggestion
```

Reply:
We shouldn't need to install again, it's already installed from conda.

Reply:
The reason it is being `pip install`ed is to test against `main`

Reply:
Yes, but we can't test against `main` in branch-21.06 anymore, only branch-21.08 should do so.

Reply:
Fair enough. Either should be fine

The primary thing is making sure we don't forget to add this back to `branch-21.08` (with `main` being used)

Reply:
Yeah, we should follow-up with a PR immediately after this lands, as we did for other libraries yesterday.